### ISSUE TITLE
Target latest Android API version

### ISSUE THREAD

**Issue by Sparow199** [niccokunzmann/mundraub-android#203]

**Comment by niccokunzmann** [comment]
Would you like to motivate how and what changing the latest API version improves?
I have Android 2.3.5 and I like to be compatible with older versions. This is my restriction.

**Comment by Sparow199** [comment]
Hello, this is an awesome article which talking about the subject.

https://www.androidpolice.com/2017/12/19/play-store-require-new-updated-apps-target-recent-api-levels-distribute-native-apps-64-bit-support/

**Comment by niccokunzmann** [comment]
Hi, thanks for that article. I only heard rumors and now there is more substance to it.

Luckily this app is published on the F-Droid App store. There is #20 which talks about publishing the app on Google Play. I will flag this as "not interesting to me" and leave it open so others can come along and discuss how to achieve compatibility with different SDKs in case they like to publish to the play store and implement that. What do you think about this?

**Comment by Sparow199** [comment]
I'm agree with you.

**Comment by niccokunzmann** [comment]
The build now releases two versions, one for old androids and one for Google Play with the latest Android version 28.
Release: https://github.com/niccokunzmann/mundraub-android/releases/tag/v1.186
![image](https://user-images.githubusercontent.com/564768/62498902-1f2bce80-b7e1-11e9-96ba-4ce47884f92d.png)

While I continue to develop for old Androids, this new APK can be used, too. It is not used by F-Droid but maybe will be used for Google Play.

**Comment by niccokunzmann** [comment]
@Sparow199 Is that like you had it in mind?

**Comment by Sparow199** [comment]
Hi, I think that in 2019, the most Android devices have 5.0+ version, so
keep developing two versions could be nightmare.

**Comment by niccokunzmann** [comment]
True, it can be. I will see if it is. It may not be so bad, since there is only one file changed. I still have an old phone so I will use the old version while trying out something new.
